en/asset/prefab.md

```diff
@@ -52,13 +52,27 @@ Icon|Description
 
 ### Add New Node
 
-A new node added under the __Prefab Instance__ will have a **+** sign in the lower right corner of the node name., and its data is stored under the __Prefab Instance__, so it will not affect the data of the associated __Prefab Asset__.
+A new node added under the __Prefab Instance__ will have a **+** sign in the lower right corner of the node name, and its data is stored under the __Prefab Instance__, so it will not affect the data of the associated __Prefab Asset__.
 
 ![prefab mounted](prefab/prefab-mounted-children.png)
 
-## Some Current Restrictions
+### Add New Component
+A new component added under the __Prefab Instance__ will have a **+** sign in the end of the node name, and its data is stored under the __Prefab Instance__, so it will not affect the data of the associated __Prefab Asset__.
+
+![](prefab/instance-add-component.png)
+
+### Remove Component
+A new component added under the __Prefab Instance__ will have a deleted data item show in the __Inspector__, its data is stored under the __Prefab Instance__, so it will not affect the data of the associated __Prefab Asset__.

```

Comment:
```suggestion
A new component added under the __Prefab Instance__ will have a deleted data item show in the __Inspector__, its data is stored under the __Prefab Instance__, it will not affect the data of the associated __Prefab Asset__.
```